src/js/test/pyodide.test.js

```diff
@@ -0,0 +1,30 @@
+import assert from "assert";
+import { loadPyodide } from "../pyodide.js";
+
+import fetch from "node-fetch";
+
+describe("Pyodide", () => {
+  let pyodide;
+  it("loadPyodide", async () => {
+    pyodide = await loadPyodide({ indexURL: "../../build/" });
+  });
+  it("runPython", async () => {
+    let res = pyodide.runPython("1+1");
+    assert.equal(res, 2);
+  });
+  it("loadPackage", async () => {
+    await pyodide.loadPackage(["micropip"]);
+  });
+  it("micropip.install", async () => {
+    // TODO: micropip currently requires a globally defined fetch function
+    global.fetch = fetch;
```

Comment:
I think we could best fix this as follows: define a js package `fetch_js` via 
```
let fetch_js_mod;
if(we are in node){
	fetch_js_mod = {
		fetch : await import("fetch")
	};
} else {
	fetch_js_mod = { fetch : globalThis.fetch };
}
pyodide.registerJsModule("fetch_js", fetch_js_mod);
```
then `from fetch_js import fetch` in `micropip` rather than from `js`.

Reply:
Yes, this is clearly for test/demo purposes only. I commented in https://github.com/pyodide/pyodide/issues/14#issuecomment-875116949 to discuss what should be the ideal long term solution. Ideally I would still love to keep this npm package as independent as possible from the rest of the Pyodide code that requires the build setup. Though last time we talked about it was difficult as far as I remeber.